scopus-import/src/test/java/no/sikt/nva/scopus/utils/ScopusGenerator.java

```diff
@@ -739,10 +740,14 @@ private Collection<? extends AuthorGroupTp> randomAuthorGroups(List<?> authorsAn
 
     private List<?> randomAuthorOrCollaborations() {
         final int maxNumbersOfAuthors = 50;
-        return IntStream.range(0, randomInteger(maxNumbersOfAuthors) + 2)
+        int randomMax = RANDOM.nextInt(2, maxNumbersOfAuthors);
```

Comment:
Generate the authors and then the collaborators as a continuation of the sequence. This isn't technically correct since the XML is not structured in this way, but there is no very simple way of ensuring that the authors/collabs will be created in a way we can compare